src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -800,30 +800,41 @@ export default class GroupComparisonStore {
         }
     });
 
+    public readonly samplesVennPartition = remoteData({
+        await:()=>[
+            this._activeGroupsNotOverlapRemoved,
+            this.sampleSet,
+            this.samples
+        ],
+        invoke:()=>{
+            const sampleSet = this.sampleSet.result!;
+            return Promise.resolve(
+                partitionCases(
+                    this._activeGroupsNotOverlapRemoved.result!,
+                    (group)=>getSampleIdentifiers([group]),
+                    (id)=>sampleSet.get({ studyId: id.studyId, sampleId: id.sampleId })!.uniqueSampleKey,
```

Comment:
strange that something called "id" is actually an object with other props.

Reply:
its an identifier, containing study and sample